Dockerfile

```diff
@@ -3,11 +3,13 @@ FROM python:3.7
 RUN groupadd --gid 10001 app && \
     useradd -g app --uid 10001 --shell /usr/sbin/nologin --create-home --home-dir /app app
 
+COPY . /app
+
+RUN chown -R 10001:10001 /app
```

Comment:
I can follow up and make the installation read-only, so we can install as root. Nothing should be writing to `/app`.